doc/articles/uno-app-solution-structure.md

```diff
@@ -0,0 +1,35 @@
+# Uno app solution structure
+
+This guide briefly explains the structure of an app created with the default [Uno app template](https://marketplace.visualstudio.com/items?itemName=nventivecorp.uno-platform-addin). It's particularly aimed at developers who have not worked with multi-platform codebases before. 
+
+## The project files in an Uno app
+
+Let's say we've created a new solution with the [Uno app template](), call it `HelloWorld`. It will already contain the following projects:
```

Comment:
`[Uno app template]()` is missing a destination hyperlink.

Reply:
```suggestion
Let's say we've created a new solution with the [Uno app template](https://marketplace.visualstudio.com/items?itemName=nventivecorp.uno-platform-addin), call it `HelloWorld`. It will already contain the following projects:
```